explorer/app/components/Table/table.tsx

```diff
@@ -71,6 +74,7 @@ export const Table = <T extends object>({
   pagination,
   sort,
   total,
+  isLoading,
```

Comment:
Would it be possible to rename this to `loading`, here and throughout the prop drilling? I would like to try to standardize booleans to the following Java-style (to maintain consistency across our projects):

Field name: `loading`
Accessor: `isLoading()`
Mutator: `setLoading(loading: boolean)`

Don't worry about other `isX` values for now (e.g. `isIdle`); we can address that in a separate PR!